Pod/Classes/PINRemoteImageManager.m

```diff
@@ -189,7 +188,7 @@ - (instancetype)initWithSessionConfiguration:(NSURLSessionConfiguration *)config
         if (!configuration) {
             configuration = [NSURLSessionConfiguration ephemeralSessionConfiguration];
         }
-        _callbackQueue = dispatch_queue_create("PINRemoteImageManagerCallbackQueue", DISPATCH_QUEUE_CONCURRENT);
+        _callbackQueue = dispatch_get_main_queue();
```

Comment:
I actually thought about this and then looked at how AFNetworking did this and I don't think supporting modification of a live queue makes sense (they just expose it as a nonatomic property). This is something that should only be set once. Or do you think there is a good reason to add extra support for this?